packages/freerdp.rb

```diff
@@ -2,29 +2,53 @@
 
 class Freerdp < Package
   description 'FreeRDP is a free implementation of the Remote Desktop Protocol.'
-  homepage 'http://www.freerdp.com/'
-  version '2.0.0-rc4'
+  homepage 'https://www.freerdp.com/'
+  version '2.0.0-rc4-1'
   source_url 'https://github.com/FreeRDP/FreeRDP/archive/2.0.0-rc4.tar.gz'
   source_sha256 '3406f3bfab63f81c1533029a5bf73949ff60f22f6e155c5a08005b8b8afe6d49'
 
-  depends_on 'ffmpeg'
-  depends_on 'gst_plugins_base'
   depends_on 'cups'
+  depends_on 'mesa'
   depends_on 'xmlto'
-  depends_on 'libxslt'
   depends_on 'ninja'
   depends_on 'xprop'
-  depends_on 'sommelier'
+  depends_on 'ffmpeg'
+  depends_on 'libsoxr'
+  depends_on 'xorg_lib'
+  depends_on 'linux_pam'
+  depends_on 'pulseaudio'
+  depends_on 'libmp3lame'
+  depends_on 'gst_plugins_base'
```

Comment:
You sure we don't need sommelier?

Reply:
Absolutely. It isn't called in the application, and users should have a choice if they want to use another window manager (`Xwayland`). I brought in all the reasonable dependencies of it, so it should build fine.

Reply:
Building successfully and actually executing are two separate things.

Reply:
🙄 It runs fine.